cpp/ql/src/Security/CWE/CWE-457/InitializationFunctions.qll

```diff
@@ -618,34 +618,43 @@ Function getAPossibleDefinition(Function undefinedFunction) {
   ) and
   result.isDefined()
 }
+ 
+private Function getTarget1(Call c) {
+  /*
+   * If there is at least one defined target after performing some simple virtual dispatch
+   * resolution, then the result is all the defined targets.
+   */
+
+  result = VirtualDispatch::getAViableTarget(c) and
+  result.isDefined()
+}
+
+private Function getTarget2(Call c) {
+  /*
+   * If we can use the heuristic matching of functions to find definitions for some of the viable
+   * targets, return those.
+   */
```

Comment:
I think the comments in `getTarget[123]` would be better placed as QLDoc on the predicates.

Reply:
Done.